lib/dynflow/execution_plan.rb

```diff
@@ -407,7 +407,7 @@ def save
 
     def self.new_from_hash(hash, world)
       check_class_matching hash
-      execution_plan_id = hash[:id]
+      execution_plan_id = hash[:uuid] || hash[:id]
```

Comment:
It's a bit strange, that `to_hash` returns `{:id => "..."}` while `new_from_hash` counts on `{:uuid => "..."}`. What about turning the `{:uuid => "..."}` into `{:id => "..."}` on persistence level so that we can still be consistent here?